api/voice-react-native-sdk.api.md

```diff
@@ -917,4 +935,8 @@ export namespace Voice {
     }
 }
 
+// Warnings were encountered during analysis:
+//
+// lib/typescript/type/CallKit.d.ts:15:9 - (ae-forgotten-export) The symbol "Constants" needs to be exported by the entry point index.d.ts

```

Comment:
@charliesantos not sure what the best solution is here. Basically, I'd like to export the `CallKit.ConfigurationOptions` type but it depends on `Constants` which is the common constants that we use for internal communication between the native and JS layers, so it includes a lot of things we probably don't want to export.

Reply:
I thought you're able to export them from here? https://github.com/twilio/twilio-voice-react-native/pull/139/files#diff-f5b962e82196241c1d8e9832b56bd5a7dff0e15f2dc9ead4d7a26916c8e957c3R12

Reply:
The issue is that each key is a constant string from the common constants file. For example, `[Constants.CallKitIconTemplateImageData]: string;` and `Constants` is something we are not and should not be exporting.

Reply:
I see. It's not ideal. Thoughts on actually using the the strings as keys? These are well defined right? Why do we need to put them in the constants file? Why not put them in the ts defs? Example
```ts
  export type ConfigurationOptions = {
    /**
     * Filename of a 80x80 PNG image that will show in the system call UI as the app icon.
     */
    callKitIconTemplateImageData: string;
//...other keys
```

Reply:
Another option is, if we really need to put them in the constants file, you might need to just stop importing the constants file in `src/type/CallKit.ts` and just "define" the actual configs as suggested above. It's better for customers but you now have two files that needs to be in sync. We can add a comment to each file as a reminder to update the other if we have to change the keys. Or another option is, don't define the config, just like what Bobie already has. But it is not good for customers..